src/lib/blueprint.ts

```diff
@@ -121,10 +123,153 @@ export interface TypesModule {
   openapi: Openapi
 }
 
-export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
-  return {
+export const createBlueprint = ({
+  openapi,
+}: {
+  openapi: Openapi
+}): Blueprint => {
+  const blueprint: Blueprint = {
     name: openapi.info.title,
     routes: [],
     resources: {},
   }
+
+  // Filter for /acs/systems/list
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
+  // Check for fake data
+  const isFakeData = openapi.info.title === 'Foo'
+
+  for (const [path, pathItem] of Object.entries(openapi.paths)) {
```

Comment:
Let's break up this function into smaller functions using this same basic functional pattern at each level (this even enables us to debug or write unit tests for each sub-function if it improves our development process). Similar to what you already did with `createResouce`. This will also let us use early return patterns instead of `continue` / `break` / `push` which is often harder to follow.

```
const createBlueprint = () => {
  const routes = Object.entries(openapi.paths).map(createRoute)
...
}

const createRoute = (openapiPath): Route => {
  const endpoints = Object.entries(pathItem).map((openapiPathItemEntry) => createEndpoint(openapiPathItemEntry, { openapiPath }))
...
}